tests/integration/test_charm.py

```diff
@@ -315,3 +316,17 @@ async def assert_links_in_configmap_by_text_value(
         assert item.text in links_texts
 
     return links_texts
+
+
+@pytest.mark.asyncio
```

Comment:
This marker is [automatically applied](https://github.com/charmed-kubernetes/pytest-operator/blob/main/docs/reference.md#pytestmarkasyncio) to any `async` function, think we can remove it.